tests/info_extraction/InfoScopeTest.py

```diff
@@ -0,0 +1,129 @@
+import os
+import unittest
+
+from coala_quickstart.info_extraction.Info import Info
+from coala_quickstart.info_extraction.InfoExtractor import InfoExtractor
+from coala_quickstart.info_extraction.InfoScope import InfoScope
+
+
+class InfoScopeTest(unittest.TestCase):
+
+    def setUp(self):
+
+        class DummyInfoExtractor(InfoExtractor):
+
+            def parse_file(self, file_content):
+                return file_content
+
+            def find_information(self, fname, parsed_file):
+                return [Info(
+                    fname,
+                    'Dummy information it is!')]
+
+        class NoInfoExtractor(InfoExtractor):
+
+            def parse_file(self, fname, file_content):
+                return file_content
+
+            def find_information(self, fname, parsed_file):
+                return []
+
```

Comment:
Why do we have these 2 classes inside `setUp` function ? Can't it be made global ?
No need to do it separately for every tests.